### PR TITLE
Removed deprecated timeout argument for Lobby calls

### PR DESCRIPTION
**Pull request by davidstl**
DO NOT MERGE UNTIL MAJOR RELEASE, 4.4

### CODE REVIEW

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs [hunk 1]
```diff
@@ -20,396 +20,8 @@ public BrainCloudProduct(BrainCloudClient client)
...
-            data[OperationParam.ProductServiceToken.Value] = token;
-
-            ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
-            ServerCall sc = new ServerCall(ServiceName.Product, ServiceOperation.GooglePlayConfirmPurchase, data, callback);
-            _client.SendRequest(sc);
-        }
-
         #region Deprecated
-        [Obsolete("Method is now available in Cloud Code only for security. If you need to use it client side, enable 'Allow Currency Calls from Client' on the brainCloud dashboard")]
+        [Obsolete("Will be removed December 2020, Method is now available in Cloud Code only for security. If you need to use it client side, enable 'Allow Currency Calls from Client' on the brainCloud dashboard")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs [hunk 2]
```diff
@@ -426,7 +38,7 @@ public BrainCloudProduct(BrainCloudClient client)
             _client.SendRequest(sc);
         }
 
-        [Obsolete("Method is now available in Cloud Code only for security. If you need to use it client side, enable 'Allow Currency Calls from Client' on the brainCloud dashboard")]
+        [Obsolete("Will be removed December 2020. Method is now available in Cloud Code only for security. If you need to use it client side, enable 'Allow Currency Calls from Client' on the brainCloud dashboard")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs [hunk 3]
```diff
@@ -443,7 +55,7 @@ public BrainCloudProduct(BrainCloudClient client)
...
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudProduct.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs [hunk 4]
```diff
@@ -162,7 +162,7 @@ public BrainCloudVirtualCurrency(BrainCloudClient client)
         }
 
         #region Obsolete
-        [Obsolete("For security reasons calling this API from the client is not recommended, and is rejected at the server by default. To over-ride, enable the 'Allow Currency Calls from Client' compatibility setting in the Design Portal.")]
+        [Obsolete("Will be removed December 2020. For security reasons calling this API from the client is not recommended, and is rejected at the server by default. To over-ride, enable the 'Allow Currency Calls from Client' compatibility setting in the Design Portal.")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs [hunk 5]
```diff
@@ -179,7 +179,7 @@ public BrainCloudVirtualCurrency(BrainCloudClient client)
             _client.SendRequest(sc);
         }
 
-        [Obsolete("For security reasons calling this API from the client is not recommended, and is rejected at the server by default. To over-ride, enable the 'Allow Currency Calls from Client' compatibility setting in the Design Portal.")]
+        [Obsolete("Will be removed December 2020. For security reasons calling this API from the client is not recommended, and is rejected at the server by default. To over-ride, enable the 'Allow Currency Calls from Client' compatibility setting in the Design Portal.")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs [hunk 6]
```diff
@@ -203,7 +203,7 @@ public JsonReader(StringBuilder input, JsonReaderSettings settings)
             /// Setting this to false, throws an exception if null is
             /// specified for a ValueType member.
             /// </remarks>
-            [Obsolete("This has been deprecated in favor of JsonReaderSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonReaderSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs [hunk 7]
```diff
@@ -213,7 +213,7 @@ public bool AllowNullValueTypes
             /// <summary>
             /// Gets and sets the property name used for type hinting.
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonReaderSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonReaderSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 8]
```diff
@@ -197,7 +197,7 @@ public JsonWriter(StringBuilder output, JsonWriterSettings settings)
             /// <summary>
             /// Gets and sets the property name used for type hinting
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 9]
```diff
@@ -207,7 +207,7 @@ public string TypeHintName
             /// <summary>
             /// Gets and sets if JSON will be formatted for human reading
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("TWill be removed December 2020. his has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 10]
```diff
@@ -217,7 +217,7 @@ public bool PrettyPrint
             /// <summary>
             /// Gets and sets the string to use for indentation
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 11]
```diff
@@ -227,7 +227,7 @@ public string Tab
             /// <summary>
             /// Gets and sets the line terminator string
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 12]
```diff
@@ -245,7 +245,7 @@ protected int Depth
             /// <summary>
             /// Gets and sets the maximum depth to be serialized
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 13]
```diff
@@ -258,7 +258,7 @@ public int MaxDepth
             /// <remarks>
             /// Respects XmlIgnoreAttribute, ...
             /// </remarks>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs [hunk 14]
```diff
@@ -268,7 +268,7 @@ public bool UseXmlSerializationAttributes
             /// <summary>
             /// Gets and sets a proxy formatter to use for DateTime serialization
             /// </summary>
-            [Obsolete("This has been deprecated in favor of JsonWriterSettings object")]
+            [Obsolete("Will be removed December 2020. This has been deprecated in favor of JsonWriterSettings object")]
```
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: This is third party. i wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs (hunk 4 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs (hunk 5 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudVirtualCurrency.cs]: Keep this, as a warning, and not preferred usage.

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs (hunk 6 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs (hunk 7 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonReader.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 8 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 9 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 10 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 11 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 12 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 13 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

File: BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs (hunk 14 above)
Comment (stevemrjones) [inline BrainCloudClient/Assets/BrainCloud/JsonFx/JsonWriter.cs]: Third Party, I wouldn't touch these

**Comment by davidstl** [comment]
This is old. We will redo this work soon. Let's redo it clean

**Pull request closed without merging**